sdk/src/wallet/operations/transaction/build_transaction.rs

```diff
@@ -27,12 +27,13 @@ where
     pub(crate) async fn build_transaction(
         &self,
         selected_transaction_data: Selected,
-        options: impl Into<Option<TransactionOptions>> + Send,
+        options: impl Into<Option<BlockOptions>> + Send,
```

Comment:
bit weird API to provide a `BlockOptions` to a method called `build_transaction`, but okay. Atm we're not using its `issuer_id` field. Is this supposed to be used when we iterate the inputs to search for an account output with a block issuer feature?